runsc/cgroup/cgroup.go

```diff
@@ -316,6 +316,7 @@ func loadPathsHelper(cgroup, mountinfo io.Reader, unified bool) (map[string]stri
 // Cgroup represents a cgroup configuration.
 type Cgroup interface {
 	Install(res *specs.LinuxResources) error
+	Set(res *specs.LinuxResources) error
```

Comment:
IMO this should be `Update()`, to make it clear that the cgroup limits are being updated after initialization. This is necessary to differentiate it from `Install()`, which also "sets" the limits upon initialization.

Reply:
makes sense, updated.